src/credentials/FabricTable.cpp

```diff
@@ -1530,6 +1530,8 @@ CHIP_ERROR FabricTable::AddNewPendingTrustedRootCert(const ByteSpan & rcac)
         mStateFlags.HasAny(StateFlags::kIsTrustedRootPending, StateFlags::kIsUpdatePending, StateFlags::kIsAddPending),
         CHIP_ERROR_INCORRECT_STATE);
 
+    ReturnErrorOnFailure(ValidateChipRCAC(rcac));
```

Comment:
Which are the possible return values here? We have to make sure that the cluster code knows how to map them to the correct NOC status

Reply:
I moved this `ValidateChipRCAC()` call to `operational-credentials-server.cpp`. 
It will return `Status::InvalidCommand` in case of failure.